packages/angular/src/schematics/application/application.ts

```diff
@@ -106,6 +106,16 @@ const nrwlHomeTemplate = {
                 <span class="gutter-left">Nrwl Connect</span>
             </a>
         </li>
+        <li class="col-span-2">
+            <a class="resource flex" href="https://nx.app/">
+                <img
+                        height="36"
+                        alt="Nx Cloud"
+                        src="https://nx.app/assets/nx-cloud-logo.svg"
+                />
+                <span class="gutter-left">Nx Cloud</span>
+            </a>
+        </li>
```

Comment:
Can you please remove the Nrwl Connect link?

Reply:
And could you add a README next to the nx-cloud-logo.svg file in the ocean repo that tells developers not to delete or change that path without adding a redirect, since it's referenced from the default template?

Reply:
@jeffbcross I wonder whether we should simply inline the svg into the generated HTML on the Nx side. That way we don't have a dependency 🤔